### ISSUE TITLE
Feat: Adding a separate blog page when the blog is clicked.

### ISSUE THREAD

**Issue by PranavKeshav24** [Nactore-Org/Nacto-Care#134]
### Describe the feature

We need to enhance our blog functionality by adding a separate blog page for each blog post. When a user clicks on a blog post, they should be directed to a dedicated page that displays the content of that specific blog, replacing the non functional blog page that we have right now. This change will ensure that when users click on a blog post, they are directed to a unique page that displays the specific content of that blog.

### Add ScreenShots

![image](https://github.com/Nactore-Org/Nacto-Care/assets/127531721/e0aecc02-d8cc-49e0-8cf0-91ffdc1e742b)
![image](https://github.com/Nactore-Org/Nacto-Care/assets/127531721/782132c2-88aa-4a2d-9486-0dab17511e77)

Please assign this issue to me. I am a contributor at GSSoC' 24. I would be very happy to work on it.

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by Harshil-Jani** [comment]
Isn't the current behaviour same ? It's just that content for all blogs is lorem epsum so you may feel it opens on same page but if we observe the URL. The slug ends with blog id.

**Comment by PranavKeshav24** [comment]
Yeah but would it not be better if we redesign the blog page, because it looks pretty simple now.

**Comment by Harshil-Jani** [comment]
Sure. Redesigning sounds like a better enhancement. However it is different from the scope of the issue. You can change the title of the issue and write some thoughts on how do you plan to re-design the blog section.

Happy to assign.

**Comment by PranavKeshav24** [comment]
@Harshil-Jani I have updated the issue title and description. Please check.
